tools/releases/templates/README

```diff
@@ -2,10 +2,8 @@ Welcome to Kuma!
 
 ===============================================================================
 
-This folder contains your download of Kuma:
-├── LICENSE
-├── NOTICE
-├── README
+This folder contains your download of Kuma:├── NOTICE
```

Comment:
I think this part`├── NOTICE` should be in a separate line

Reply:
Good catch, fixed in last commit.